build.gradle

```diff
@@ -54,6 +55,8 @@ dependencies {
     compile "com.facebook.presto:presto-spi:${prestoVersion}"
     compile "com.facebook.presto:presto-common:${prestoVersion}"
 
+    runtimeOnly group: 'io.netty', name: 'netty-all', version:"{nettyVersion}"
```

Comment:
i think maybe this needs to be compile ?  didn't seem to work for me w/ runtimeOnly for whatever reason

Reply:
Changed to 'compile'